code/modules/shuttle/supply.dm

```diff
@@ -107,8 +107,11 @@ GLOBAL_LIST_INIT(blacklisted_cargo_types, typecacheof(list(
 	var/msg = ""
 	var/sold_atoms = ""
 
+	for(var/obj/mecha/ME in areaInstance)
+		ME.wreckage = null // So mechs don't explode when they arrive at centcom
```

Comment:
This feels kind of snowflaky here but I tried making the mecha export datums all within a subtype which did this on sell_object() but it never worked in testing. Or I fucked something up in my sleep deprived state.